docs/Configuration.md

```diff
@@ -40,22 +40,22 @@ the [Zonemaster Engine default profile].
 There is a `default` profile that is special.
 It is always available.
 If it is not explicitly mapped to a profile JSON file, it is implicitly
-mapped to the Zonemaster Engine default profile.
+mapped to the [Zonemaster Engine default profile].
 
 Specifying a profile JSON file that contains a complete set of profile
 data is equivalent to specifying a profile JSON file with only the parts
-that differ from the Zonemaster Engine default profile.
+that differ from the [Zonemaster Engine default profile].
 Specifying a profile JSON file that contains no profile data is equivalent
-to specifying a profile JSON file containing the entire Zonemaster Engine
-default profile.
+to specifying a profile JSON file containing the entire
+[Zonemaster Engine default profile.]
 
 ## ZONEMASTER section
 
 TBD
 
 --------
 
-[Profile JSON files]: https://metacpan.org/pod/Zonemaster::Engine::Config#PROFILE-DATA
+[Profile JSON files]: https://github.com/zonemaster/zonemaster-engine/blob/master/docs/Profiles.md
 [Profile names]: API.md#profile-name
 [Profiles]: Architecture.md#profile
-[Zonemaster Engine default profile]: https://metacpan.org/pod/Zonemaster::Engine::Config#DESCRIPTION
+[Zonemaster Engine default profile]: https://github.com/zonemaster/zonemaster-engine/blob/master/docs/Profiles.md
```

Comment:
The above link is the same as the earlier one. The default profile link in the page "https://github.com/zonemaster/zonemaster-engine/blob/master/share/default.profile" returns a 404 error

Reply:
Please see https://github.com/zonemaster/zonemaster-engine/pull/687